source/extensions/transport_sockets/tls/context_impl.cc

```diff
@@ -1362,6 +1385,31 @@ ServerContextImpl::selectTlsContext(const SSL_CLIENT_HELLO* ssl_client_hello) {
   return ssl_select_cert_success;
 }
 
+ValidationResults ContextImpl::customVerifyCertChainForQuic(
```

Comment:
Why moving around this?

Reply:
Some code moved to a different namespace. To make the interfaces work out correctly, `TlsContext` moved from this extensions's namespace to `Envoy::Ssl`. So all the member functions moved to a different namespace block in the file, hence all the movement.